docs/install/README.md

```diff
@@ -835,7 +835,7 @@ on Docker Hub for a list of available images.
 Install with kubectl:
 
 ```console
-kubectl apply -k github.com/k8ssandra/k8ssandra-operator/config/deployments/control-plane
+kubectl apply --server-side -k github.com/k8ssandra/k8ssandra-operator/config/deployments/control-plane\?ref\=v1.0.0
```

Comment:
I'm getting this error when I try to install with this command:

```
% kubectl apply --server-side -k github.com/k8ssandra/k8ssandra-operator/config/deployments/control-plane\?ref\=v1.0.0
error: accumulating components: accumulateDirectory: "couldn't make target for path '/private/var/folders/z0/1j_k5l190v50n6gxfy0yxczc0000gn/T/kustomize-263141531/config/components/single-namespace': json: unknown field \"replacements\""
```

Any idea what's up with that?

Reply:
You need to use kustomize build ... | kubectl apply --server-side  -f -